.azure/pipelines/jobs/default-build.yml

```diff
@@ -75,182 +65,158 @@ parameters:
   cancelTimeoutInMinutes: 15
 
 jobs:
-- job: ${{ coalesce(parameters.jobName, parameters.agentOs) }}
-  displayName: ${{ coalesce(parameters.jobDisplayName, parameters.agentOs) }}
-  dependsOn: ${{ parameters.dependsOn }}
-  ${{ if ne(parameters.condition, '') }}:
-    condition: ${{ parameters.condition }}
-  timeoutInMinutes: ${{ parameters.timeoutInMinutes }}
-  workspace:
-    clean: all
-  strategy:
-    ${{ if ne(parameters.matrix, '') }}:
-      maxParallel: 8
-      matrix: ${{ parameters.matrix }}
-  # Map friendly OS names to the right queue
-  # See https://github.com/dotnet/arcade/blob/master/Documentation/ChoosingAMachinePool.md
-  pool:
-    ${{ if ne(parameters.poolVmImage, '') }}:
-      vmImage: ${{ parameters.poolVmImage }}
-    ${{ if and(eq(parameters.poolVmImage, ''), eq(parameters.agentOs, 'macOS')) }}:
-      vmImage: macOS-10.13
-    ${{ if and(eq(parameters.poolVmImage, ''), eq(parameters.agentOs, 'Linux')) }}:
-      vmImage: ubuntu-16.04
-    ${{ if ne(parameters.poolName, '') }}:
-      name: ${{ parameters.poolName }}
-    ${{ if and(eq(parameters.poolName, ''), eq(parameters.agentOs, 'Windows')) }}:
-      ${{ if eq(variables['System.TeamProject'], 'public') }}:
-        name: NetCorePublic-Pool
-        ${{ if ne(parameters.isTestingJob, true) }}:
-          # Visual Studio Build Tools
-          queue: BuildPool.Windows.10.Amd64.VS2019.BT.Open
-        ${{ if eq(parameters.isTestingJob, true) }}:
+- template: /eng/common/templates/job/job.yml
+  parameters:
+    name: ${{ coalesce(parameters.jobName, parameters.agentOs) }}
```

Comment:
When I used `enablePublishBuildArtifacts`, it seemed `$(Agent.JobName)` matched `displayName` instead of `name`. Thus, I'm worried about connections made in codesign-xplat and the BAR publication step. Is this setup invalid?